nodes/src/nodes/tool_github/IInstance.py

```
@@ -0,0 +1,1034 @@
+
+    # -----------------------------------------------------------------------
+    # Helpers
+    # -----------------------------------------------------------------------
+
+    def _token(self) -> str:
+        return self.IGlobal.token
+
+    def _repo(self, args: dict) -> str:
+        repo = (args.get('repo') or self.IGlobal.default_repo or '').strip()
+        if not repo:
+            raise ValueError('repo is required (or configure a default repo in the node settings)')
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)